app/lib/theme/widgets/FileWidget.tsx

```diff
@@ -178,7 +178,7 @@ const FileWidget: React.FC<WidgetProps> = ({
     const variables = {
       input: {
         attachmentPatch: {
-          isDeleted: true,
+          archivedAt: new Date().toISOString(),
```

Comment:
toISOString The timezone is always zero UTC offset. Are we string dates in UTC? 
... just want to make sure that we are nor required to use PST(PDT)